### DISCUSSION TITLE
Is there any way to put link/button within Select Content

### DISCUSSION THREAD

**Discussion by AayushKarki714** [radix-ui/primitives#2701] [Help]
I have a design to replicate. All of the things seems to be fine. but the select content  has a link in the bottom. Is there any way of adding link within select content. It also need's to be accessible through keyboard. 

Here is the design 

![image](https://github.com/radix-ui/primitives/assets/99877352/d704c87e-1dff-4dd4-9f75-e99816f74dff)

Manage categories need's to be a link and It shouldn't update the select state when clicked

**Comment by benoitgrelard** [comment]
This doesn't look like a select as the value isn't represented in the trigger…

**Reply by AayushKarki714** [reply]
The value does gets reflected in the trigger. Here I have an exception that the very first item of the select content should use category as a value in select trigger and for all of the remaining item it work's just like radix select. The issue I am facing is I have got some button/link within select content and it needs to accessible through keyboard and shouldn't change the value of select trigger. It also needs to close the select content when pressed

**Reply by benoitgrelard** [reply]
What should happen once that button is activated?

**Reply by AayushKarki714** [reply]
It usually opens a radix dialog. It could be easily replicated using radix dropdown. But I was skeptical about the accessibility part of it.
